config/initializers/env_var_constants.rb

```diff
@@ -11,4 +11,5 @@
 # Definition of fund
 FUND_FULL = ENV['DARIA_FUND_FULL'] || 'DC Abortion Fund'
 FUND_DOMAIN = FUND_FULL.gsub(' ', '').downcase
+FUND_PHONE = ENV['DARIA_FUND_PHONE'] || '202-452-7464'
```

Comment:
Do we want it to default to our number? It seems like ours is the _least_ likely not have the ENV vars setup :smile: 

Not sure what a good alternative would be, though. Is there any sort of national registry or AF info line?

Reply:
Haha, I definitely hear you. It kinda fits with the existing pattern of DCAF as the primary use case we work against, and other funds as 'eh, throw it in an env var if you need something different'. (Whether that is a good pattern is probably questionable.)